.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
remove stale comment or comment about cypress/usage of this workflow?

Reply:
Removed